bin/nrnivmodl.in

```diff
@@ -118,21 +139,26 @@ for i in "${files[@]}" ; do
   base_names+=( "$(basename "$i" .mod)" )
 done
 
-echo -n "Mod files:"
 for i in "${files[@]}" ; do
-  dir_name=$(dirname "$i")
-  echo -n " \"$dir_name/${i%.mod}.mod\""
+  echo  "- ${i%.mod}.mod"
 done
-echo ""
-echo ""
-
 
 if [ ! -d "$MODSUBDIR" ] ; then
   echo "Creating '$MODSUBDIR' directory for .o files."
   echo
-  mkdir "$MODSUBDIR"
+  mkdir -p "$MODSUBDIR"
+fi
+
+if [ "$LinkCoreNEURON" = true ] && [ "${NMODL}" ] ; then
```

Comment:
@alkino: may be you can remind me: we were saying we will add a cli option to `nrnivmodl` to be more explicit control?